### ISSUE TITLE
N64 Advanced RGB 15Bit color issue

### ISSUE THREAD

**Issue by JeromeMarousez** [borti4938/n64rgb#8]
EDIT : I changed the title to help people find this post in the future. This is NOT an issue with the 15Bit color but looks like it at first. 

Hi Borti, I've installed the board and I have an issue where it looks like the graphics put themselves in 15bit mode only just way worse. I have huge bending issues (looks like contrast and saturation are too high too) and when that happens, the OSD isn't accessible anymore. Sometimes, it appears off screen (top left corner) and seems to be having issues of its own (meaning that I see the bottom part of the OSD but text that should be in the middle of the OSD).
Current firmware:
HDL  1.55
SW   1.56
Short video of the installation : https://www.youtube.com/watch?v=tf5GaEPqcM0
I went over my work multiple times but now I'm suspecting the board/firmware (which I don't like to do but I'm out of options and the OSD going nuts makes me think that the console has nothing to do with it). The board has been assembled and flashed by Bad_Ad84. The console is PAL CPU-01. I've seen the console work, in fact, I thought that I have solved the issue and ran it for several hours again to see the issue come back again. I've soaked the board in 99.9% IPA many, many times to make sure my flux wasn't the issue.

Any idea ? Thanks in advance !

**Comment by JeromeMarousez** [comment]
More on that story : With my PVM, I can use the under scan feature and clearly see that the OSD is BEHIND the picture outputted by the console (sometimes, other times it doesn't appear at all or does strange things). Apart from the weird color issue, (I'm using Mario 64 btw - other games do the same), mario is still looking like someone messed up with photoshop filters but the picture is fine, not flickery or anything. Sometimes, the OSD comes back, blinks and is split in half, when that happens, it doesn't register my inputs properly or at all.

I've updated the firmware to 1.58 and it's still the same.

Get that, on Mario menu screen, I see Mario's hand (the OSD is there, just behind the picture), no matter how many times I press B, the OSD won't leave. And when I press A, the hand pushes the square which should start the game and it doesn't ! The square is being pushed though, but nothing happens. Somehow, the Advanced RGB board prevents me from doing things in the game ?!! How weird is that =) The console knows that I pressed A, because the square reacts to it but the game doesn't start ?!

That's beyond me =)
